src/app/__init__.py

```diff
@@ -1,9 +1,10 @@
 
+import os
```

Comment:
Nice. [This resource](https://pep8.org/#imports) may be a good reference for the future.